_posts/2015-07-09-open-code-of-conduct.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Announcing the Open Code of Conduct
+author: bkeepers
```

Comment:
@caniszczyk you should probably take authorship for this. Or we can update the template to allow no authorship.